src/libraries/System.Reflection.Emit/ref/System.Reflection.Emit.cs

```diff
@@ -19,9 +19,9 @@ public sealed partial class AssemblyBuilder : System.Reflection.Assembly
         public override string Location { get { throw null; } }
         public override System.Reflection.Module ManifestModule { get { throw null; } }
         public override bool ReflectionOnly { get { throw null; } }
-        [System.Diagnostics.CodeAnalysis.RequiresDynamicCode("Generating new code at runtime is not supported with native AOT.")]
+        [System.Diagnostics.CodeAnalysis.RequiresDynamicCode("Creating a DynamicMethod requires dynamic code.")]
         public static System.Reflection.Emit.AssemblyBuilder DefineDynamicAssembly(System.Reflection.AssemblyName name, System.Reflection.Emit.AssemblyBuilderAccess access) { throw null; }
-        [System.Diagnostics.CodeAnalysis.RequiresDynamicCode("Generating new code at runtime is not supported with native AOT.")]
+        [System.Diagnostics.CodeAnalysis.RequiresDynamicCode("Creating a DynamicMethod requires dynamic code.")]
```

Comment:
These two aren't for creating a DynamicMethod:

```suggestion
        [System.Diagnostics.CodeAnalysis.RequiresDynamicCode("Defining a dynamic assembly requires dynamic code.")]
```